app/views/includes/document-upload.html

```diff
@@ -6,9 +6,42 @@
             <p class="document-upload__dropzone-text bold">{{ common.or }}</p>
             <p class="document-upload__dropzone-text">{{ common.documentUploadDragAndDrop }}</p>
         </div>
+        <div class="document-upload__fallback fallback">
+            <form class="form" method="post" action="/upload-document/add" novalidate>
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+                <div class="form-group">
+                    <label for="file" class="visually-hidden">{{ content.title }}</label>
+                    <input name="file" type="file" id="file">
+                </div>
+                <div class="form-group">
+                    <input id="button" class="button-secondary" type="submit" role="button" value="{{ common.documentUploadUploadDocument }}">
```

Comment:
Loving the variable name `documentUploadUploadDocument`

Reply:
Yep it's special...in a later PR it gets refactored to include a hyphen too...something exciting to watch out for 😄

Reply:
👍